microshift.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+source tools.sh
+source snc-library.sh
+
+BUNDLE_TYPE="microshift"
+INSTALL_DIR=crc-tmp-install-data
+CRC_VM_NAME=${CRC_VM_NAME:-crc}
+BASE_DOMAIN=${CRC_BASE_DOMAIN:-testing}
+MIRROR=${MIRROR:-https://mirror.openshift.com/pub/openshift-v4/$ARCH/clients/ocp}
+OPENSHIFT_RELEASE_VERSION=4.12.0
+
+if grep -q -i "release 8" /etc/redhat-release
+then
+  echo "Check if system is registered"
+  # Check the subscription status and register if necessary
+  if ! sudo subscription-manager status >& /dev/null ; then
+     sudo subscription-manager register
+  fi
+else
+  echo "This script only works for RHEL-8"
```

Comment:
Isn't this missing an `exit`?
I'd "unroll" the test, `if !running_on_rhel8; then echo "only works on RHEL-8"; exit; fi`
and then `if ! subscription-manager status; then echo "machine must be registered"; exit; fi"`

Reply:
Done.